pandas/_libs/lib.pyx

```diff
@@ -2315,3 +2312,119 @@ def fast_multiget(dict mapping, ndarray keys, default=np.nan):
             output[i] = default
 
     return maybe_convert_objects(output)
+
+
+@cython.wraparound(False)
```

Comment:
I'd like to move these added routines to parsing.pyx; these are not useful outside of the parser and thus should be colocated.

Reply:
Done